### PR TITLE
add browserSetup fix for environemental setup

### PR DESCRIPTION
**Pull request by Winner95**
In some cases we want  to run config.browserSetup before config.getSections

### CODE REVIEW

**Comment by Winner95** [comment]
Updated.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v0.0.33-alhpa
+* Move `browserSetup` function call to constructor phase in hot-index lifecycle.
```
Comment (kdzuin) [inline CHANGELOG.md]: could you please explain 'what' not 'how'? like 'Fix race condition for browserSetup being called after rendering styleguide'

Comment (kdzuin) [reply CHANGELOG.md]: could you please update package.json as well?

Comment (Winner95) [reply CHANGELOG.md]: yes, I added update in this pull request.

**Pull request merged**